godtools/Views/TractElements/Header.swift

```diff
@@ -26,6 +26,8 @@ class Header: BaseTractElement {
         }
     }
     
+    var includesNumber = false
```

Comment:
Okay cool. That's okay if it's used in a child element. I just missed that during review.